config/robotframework_aio/release_items_Robotframework_AIO.json

```diff
@@ -214,9 +214,14 @@
                               ],
                               "0.13.1." : [
 "
-**Robot Framework**
+**Robot Framework pipeline**
 
 * Added test coverage to ensure functionality and improve overall code reliability
+
+**VSCodium Package**
+
+* Replace `robotframework-lsp <https://github.com/robocorp/robotframework-lsp>`_ extension 
```

Comment:
Hi Ngoan,
very good!
Please add that robotframework-lsp is deprecated.
Thank you,
 Thomas

Reply:
Hi Thomas,

I added the deprecated information and referent link to issue (notification) on robotframework-lsp repo in new commit.
Please review and give your feeling about this change.

Thank you,
Ngoan